ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/ModelServer.resource

```diff
@@ -108,16 +114,16 @@ Set Number of GPU With Buttons
             Click GPU Minus Button
         END
     END
-    ${current}=    Get Element Attribute    ${GPU_SECTION_INPUT}    value
+    ${current}=    Get Element Attribute    ${SERVING_ACCELERATOR_INPUT_XPATH}    value
     Should Be Equal As Integers    ${current}    ${no_gpus}
 
 Click GPU Plus Button
     [Documentation]    Click the plus button in the GPU selector
-    Click Element    ${GPU_SECTION_PLUS}
+    Click Element    ${SERVING_ACCELERATOR_PLUS_BUTTON_XPATH}
 
 Click GPU Minus Button
     [Documentation]    Click the minus button in the GPU selector
-    Click Element    ${GPU_SECTION_MINUS}
+    Click Element    ${SERVING_ACCELERATOR_LESS_BUTTON_XPATH}
```

Comment:
Nitpick - I would vote to use either `plus` and `minus` OR `more` and `less` naming :upside_down_face: .

Reply:
Fixed in [58f7086](https://github.com/red-hat-data-services/ods-ci/pull/971/commits/58f70868872c278508328aff89deadb11a0cc419)